.changeset/six-paths-speak.md

```diff
@@ -0,0 +1,4 @@
+---
+---
```

Comment:
Werkt dit? zonder semver versie? Lijkt me dat als we een changelog toevoegen er ook, op z'n minst, een `patch` bij moet? of gaat dat enkel over functionaliteit?